lib/components/narrative/realtime-annotation.js

```diff
@@ -55,7 +61,9 @@ export default class RealtimeAnnotation extends Component {
             className='toggle-realtime'
             onClick={toggleRealtime}
           >
-            {useRealtime ? `Ignore` : `Apply`} service delays
+            <FormattedMessage
+              id={`components.RealtimeAnnotation.${useRealtime ? 'ignore' : 'apply'}ServiceDelays`}
+            />
```

Comment:
The [`@formatjs/cli`](https://formatjs.io/docs/tooling/cli) tool (which we may want to use in the future) isn't able to parse dynamically generated message IDs. So I recommend separating this out as follows:

```suggestion
            {useRealtime 
              ? <FormattedMessage id='components.RealtimeAnnotation.ignoreServiceDelays' />
              : <FormattedMessage id='components.RealtimeAnnotation.applyServiceDelays' />
            }
```

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/b28ccbdecf85d6e381b8a03c27b0e2910bda849f